os/kernel/task/task_create.c

```diff
@@ -284,6 +298,12 @@ static int thread_create(FAR const char *name, uint8_t ttype, int priority, int
 #ifndef CONFIG_BUILD_KERNEL
 int task_create(FAR const char *name, int priority, int stack_size, main_t entry, FAR char *const argv[])
 {
+#ifdef CONFIG_BINARY_MANAGER
+	if (BM_PRIORITY_MIN <= priority && priority <= BM_PRIORITY_MAX) {
+		bmdbg("Invalid priority %d, it should be lower than %d or higher than %d\n", priority, BM_PRIORITY_MIN, BM_PRIORITY_MAX);
```

Comment:
When debugging from User side, user will be enable the sdbg, not bmdbg. So I think change from bmdbg to sdbg is better.